resources/git-hooks/commit-msg

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+#
+# This commit-msg hook validates commit messages
+# to ensure that they meet our conventional commit standard.
+#
+#
+#   "type: message"
+#   "type(scope): message"
+#   Optionally, append ". Fixes #1234"
+#
+# Reference: https://github.com/keymanapp/keyman/wiki/Pull-Request-and-Commit-workflow-notes
+#
+
+#
+# Configurable options for types, scopes and message length.
+#
+
+types=(fix feat chore change docs style refactor test)
+scopes=(android ci common developer ios linux mac web windows)
```

Comment:
A couple questions about scopes.

1. do we lump `lm-layer` with `web`?
2. what about `keyboardprocessor`, `oem`, and `resources`?

Reply:
* Yes, `lm-layer` goes with `web` (if you see the label in GitHub, it is now `web-lm-layer`).
* `keyboardprocessor` and `resources` go in `common`. 
* For `oem` use the platform that is being updated. If multiple platforms, you may want to go with `common`

I've added most of this to the wiki.